gamma correct colorpicker image panel

sjpt opened the pull request.

sjpt committed:
updates for later versions of three.js

sjpt committed:
build of support for later three.js

sjpt committed:
update torus example for webXR

sjpt committed:
add original veview for comparison (not used)

sjpt committed:
remove unnecessary repeat of renderer.setAnimationLoop, and return of animate from vrviewer

sjpt committed:
illustration of scale effecting blur

sjpt committed:
remove some 'noise' from blur
compensate antialias for scale (not quite right, but much better)

sjpt committed:
modify (correct?) vScale calculation

sjpt committed:
Use Layout.TEXT_SCALE in shader

sjpt committed:
remove support with no standard derivatives. For now just comment out dead (vScale etc) code, next revision remove it completely

sjpt committed:
version with non-derivative path and vScale removed (were commented out in previous version)

sjpt committed:
return mouseInput.intersections; from update(), so datguix can use it for selection

sjpt committed:
allow custom button colours

sjpt committed:
do not update geometry for unchanged text (especially slider values)

sjpt committed:
comment out stopImmediatePropagation for window mousedown

sjpt committed:
make sure w value set for colorpicker shaders

sjpt committed:
change after npm update

sjpt committed:
fix colourpicker bug

sjpt committed:
windows buils
built to match Organic changes, three revision 151
SubdivisionModifier used more consistently

sjpt committed:
more reduce interaction between makeDraggable drag and this drag

sjpt committed:
Merge remote-tracking branch 'origin/master'

sjpt committed:
update core-js to fix warning

sjpt committed:
gamma correct the colour panels

xinaesthete commented:
Sounds like an improvement. Remind me to look more properly if I don't get around to merging soon.

Also take a look at [oklab](https://bottosson.github.io/posts/oklab/) - seems like quite a useful/interesting color-space...

sjpt committed:
colourpicker uses sqrt for approx colour correction
folder.js uses simpler z position code, which an prevent folder lockout when too many folders undocked
larger MAX_FOLDER_ITEMS_IN_COLUMN = 40

xinaesthete commented:
Should be ok, bad gamma of that was annoying I agree. I might want to check a few bits with other changes to folders etc.

sjpt committed:
Merge branch 'master' of https://github.com/xinaesthete/dat.guiVR

# Conflicts:
#	build/datguivr.js
#	build/datguivr.map
#	build/datguivr.min.js
#	modules/datguivr/colorpicker.js

sjpt committed:
remove blank lines, may help merge .... confused
